models/fixtures/user.yml

```diff
@@ -218,3 +218,33 @@
   avatar_email: user13@example.com
   num_repos: 3
   is_active: true
+
```

Comment:
Didn't want to intervene with other test cases because of duplicate email address.
But if that's not the case of course I can remove user16.